database.env

```diff
@@ -1,3 +1,3 @@
 POSTGRES_USER=postgres
 POSTGRES_PASSWORD=mypassword
-POSTGRES_DB=entitycorex
\ No newline at end of file
+POSTGRES_DB=housinginfo-test
```

Comment:
Although we don't really yet use a local development postgres service while working on these projects, technically the same `database.env` file contents here are used during setup of both the test and dev databases. (This file is referenced in both services in `docker-compose.yml`.)

This means that we might not want to append `-test` to the database name - or alternatively, in future, we might want to use a different file for the dev database's setup. In which case, no changes to make here now, and perhaps instead an update to the base-api for us to consider in future.